extensions/sequel-ace/src/database_list.tsx

```diff
@@ -39,7 +39,7 @@ export default function DatabaseList() {
       <List.EmptyView icon={{ source: "no-view.png" }} title="No databases found in Sequel Ace, go add one!" />
       {state &&
         state.grps?.map((item) => {
-          const subtitle = `${item.connections.length}` + ` item${item.connections.length > 1 ? "s" : ""}`;
+          const subtitle = `${item.connections.length}` + ` item${item.connections.length !== 1 ? "s" : ""}`;
```

Comment:
why 'item.connections.length !== 1' should be better than > `item.connections.length > 1`?